munit/shared/src/main/scala/munit/MUnitRunner.scala

```diff
@@ -325,7 +325,7 @@ class MUnitRunner(val cls: Class[_ <: Suite], newInstance: () => Suite)
       catch onError
     result.map { _ =>
       notifier.fireTestFinished(description)
-      true
+      !test.tags(Pending)
```

Comment:
Yes, PendingComment uses Pending (because `def pending(comment: String): TestOptions = pending.tag(PendingComment(comment))`), so we only need to check for Pending.